Cargo.toml

```diff
@@ -136,6 +139,7 @@ gui = ["win_manifest","kanata-parser/gui",
   "native-windows-gui/tray-notification","native-windows-gui/message-window","native-windows-gui/menu","native-windows-gui/cursor","native-windows-gui/high-dpi","native-windows-gui/embed-resource","native-windows-gui/image-decoder","native-windows-gui/notice","native-windows-gui/animation-timer",
 ]
 zippychord = ["kanata-parser/zippychord"]
+watch = ["notify", "notify-debouncer-mini", "tempfile"]
```

Comment:
Seems reasonable for `watch` to be a default feature, let's add it to `default`.